### ISSUE TITLE
Can this be implemented in Ionic 1

### ISSUE THREAD

**Issue by surajde16** [viithiisys/ionic-location-sharing#1]
Hello,
Thank for great work on location sharing like WhatsApp.
I just want to know if this can be implemented in Ionic version 1. Currently, we are using https://github.com/apache/cordova-plugin-geolocation this as the plugin to get lat and long points.

Sometimes in remote places, we are unable to get accurate lat-long points and sometimes we fetch wired locations at least as far as 1000 or 2000 kms away distances. 
Just like how we get from WhatsApp, I want to implement same in my app.
Basically am very much concerned about accuracy.

Any help on this will be a great help.

Thank you
